tests/functional/strategy/test_config.py

```diff
@@ -33,6 +33,13 @@ def test_strategy_deployment(strategist, vault, TestStrategy):
     assert not strategy.tendTrigger(0)
 
 
+def test_strategy_no_reinit(strategist, vault, TestStrategy):
+    strategy = strategist.deploy(TestStrategy, vault)
+
+    with brownie.reverts("Strategy already initialized"):
+        strategy.initialize(vault, strategist, strategist, strategist)
+
```

Comment:
add a test here deploying a new proxy-version from the already deployed version, and show re-init doesn't work there either.

Reply:
I was preparing it, let's see the next commit